spring-boot-tools/spring-boot-maven-plugin/src/main/java/org/springframework/boot/maven/AbstractRunMojo.java

```diff
@@ -68,22 +68,22 @@
 	 * this feature and many more.
 	 * @since 1.0
 	 */
-	@Parameter(property = "run.addResources", defaultValue = "false")
+	@Parameter(property = "spring-boot.run.addResources", defaultValue = "false")
```

Comment:
I know this goes against what is stated in the original issue but I am wondering if this should be changed. The fact it starts with `run` is _a_ qualifier on its own. 

Changing this will impact users for sure so I am wondering if we should do it.